### ISSUE TITLE
[BUG] Invisible Icons/Items - Incompatibility with Sodium

### ISSUE THREAD

**Issue by SirOMGitsYOU** [shedaniel/Materialisation#55]
Bug when using Materialisation in combination with Sodium

![Bug in action](https://i.imgur.com/eDSsc11.jpeg)

**Comment by Squiffilect** [comment]
Getting the same when used with the Everything Fabric pack
![8u242-b08 Screenshot 2021 02 19 - 17 55 12 13](https://user-images.githubusercontent.com/52787861/108574395-31ac2100-730f-11eb-929a-c850c99e4de2.png)

**Comment by SirOMGitsYOU** [comment]
> Getting the same when used with the Everything Fabric pack
> ![8u242-b08 Screenshot 2021 02 19 - 17 55 12 13](https://user-images.githubusercontent.com/52787861/108574395-31ac2100-730f-11eb-929a-c850c99e4de2.png)

Yeah that’s my modpack 😃 I am just reporting this here to hopefully have a fix as ideally would like to keep sodium for the performance benefits
